[skip-ci] Replace coordinate map in Kibana getting started docs with Maps

nreese opened the pull request:
This PR updates the Kibana getting started docs. It replaces coordinate map with Maps, https://www.elastic.co/guide/en/kibana/current/tutorial-visualizing.html#tutorial-visualize-map.

elasticmachine commented:
Pinging @elastic/kibana-gis (Team:Geo)

elasticmachine commented:
Pinging @elastic/kibana-docs (Team:Docs)

gchaps commented on docs/getting-started/tutorial-visualizing.asciidoc:
```diff
@@ -141,11 +140,15 @@ You haven't defined any buckets yet, so the visualization is a map of the world.
-.. In the *Buckets* pane, click *Add > Geo coordinates*.
-.. Set *Aggregation* to *Geohash*.
-.. Set *Field* to *geo.coordinates*.
+.. Click *Add layer* button.
```
```suggestion
.. Click *Add layer*.
```

gchaps commented on docs/getting-started/tutorial-visualizing.asciidoc:
```diff
@@ -141,11 +140,15 @@ You haven't defined any buckets yet, so the visualization is a map of the world.
+.. Click the *Add layer* button.
 
-. Click *Apply changes* image:images/apply-changes-button.png[].
+. Set the vector style.
```
The text I see in the UI is "Layer style".  Would it be better to say "Set the layer style"?

gchaps commented on docs/getting-started/tutorial-visualizing.asciidoc:
```diff
@@ -141,11 +140,15 @@ You haven't defined any buckets yet, so the visualization is a map of the world.
+.. For *Border color*, select white.
+.. Click *Save & close*.
 +
 The map now looks like this:
```
I had to click "Refresh" for the data to show.

nreese replied:
style settings should be applied immediately

gchaps commented on docs/getting-started/tutorial-visualizing.asciidoc:
```diff
@@ -154,15 +157,17 @@ image::images/tutorial-visualize-map-2.png[]
 . Navigate the map by clicking and dragging.  Use the controls
 on the left to zoom the map and set filters.
-. *Save* this map with the name `Map Example`.
+. In the application toolbar, click *Save*.
```
Elsewhere in the tutorial, we shorten this a bit.  For example:

To save this chart so you can use it later, click *Save* in
the top menu bar and enter `Pie Example`.

gchaps commented on docs/getting-started/tutorial-visualizing.asciidoc:
```diff
@@ -154,15 +157,17 @@ image::images/tutorial-visualize-map-2.png[]
 The final visualization is a Markdown widget that renders formatted text.
 
-. Create a *Markdown* visualization.
+. Return to the Visualization app and create a *Markdown* visualization.
```
It might be better to change the order.  That is create the markdown before the map so the user doesn't have to return to the Visualization app.

nreese commented:
> When I added the map to the dashboard, it did not have any data. Is it because the time filter is set to the last 15 minutes?

Yes, the dashboard time picker will need to be set to the time range.

nreese commented:
> Do we need to remove the word "coordinate" from this bulleted list

Thanks, updated text in list

> I needed to click Refresh after I changed the time filter in the dashboard

Thanks, added step about clicking `Update` button when setting time filter

> My map doesn't look like what I see in the tutorial

I had zoomed the map in to make a more interesting screen shot. Should I just use the one you provided?
